packages/template-retail-react-app/app/pages/cart/index.jsx

```diff
@@ -56,22 +57,23 @@ import {
     useShopperCustomersMutation
 } from '@salesforce/commerce-sdk-react'
 import {useCurrentCustomer} from '@salesforce/retail-react-app/app/hooks/use-current-customer'
+import UnavailableProductConfirmationModal from '@salesforce/retail-react-app/app/components/unavailable-product-confirmation-modal'
 
+const DEBOUNCE_WAIT = 750
```

Comment:
It's already in develop, and I get that we just put a hardcoded variable into a constant but we might want to put a comment on here to say why it's 750 at some point